ui/admin/tests/acceptance/roles/global-scope-test.js

```diff
@@ -509,6 +514,72 @@ module('Acceptance | roles | global-scope', function (hooks) {
     assert.strictEqual(currentURL(), urls.manageCustomScopes);
   });
 
+  test('user can choose to only deselect org using modal on manage custom scopes page', async function (assert) {
+    instances.role.update({
+      grant_scope_ids: [instances.scopes.org.id, instances.scopes.project.id],
+    });
+    await visit(urls.manageScopes);
+
+    await click(`[href="${urls.manageCustomScopes}"]`);
+    await click(SCOPE_CHECKBOX_SELECTOR('org', instances.scopes.org.id));
+
+    assert.dom(REMOVE_ORG_MODAL).isVisible();
+
+    await click(REMOVE_ORG_ONLY_BUTTON);
+    await click(SAVE_BTN_SELECTOR);
+
+    assert.strictEqual(currentURL(), urls.manageScopes);
+    assert.dom(BUTTON_ICON_SELECTOR).isVisible();
+
+    await click(SAVE_BTN_SELECTOR);
+
+    assert.strictEqual(currentURL(), urls.roleScopes);
+    assert.strictEqual(findAll(TABLE_ROW_SELECTOR).length, 1);
+  });
+
+  test('user can choose to deselect org and projects using modal on manage custom scopes page', async function (assert) {
+    instances.role.update({
+      grant_scope_ids: [instances.scopes.org.id, instances.scopes.project.id],
+    });
+    await visit(urls.manageScopes);
+
+    await click(`[href="${urls.manageCustomScopes}"]`);
+    await click(SCOPE_CHECKBOX_SELECTOR('org', instances.scopes.org.id));
+
+    assert.dom(REMOVE_ORG_MODAL).isVisible();
+
+    await click(REMOVE_ORG_PROJECTS_BUTTON);
+    await click(SAVE_BTN_SELECTOR);
+
+    assert.strictEqual(currentURL(), urls.manageScopes);
+    assert.dom(BUTTON_ICON_SELECTOR).isVisible();
+
+    await click(SAVE_BTN_SELECTOR);
+
+    assert.strictEqual(currentURL(), urls.roleScopes);
+    assert.strictEqual(findAll(TABLE_ROW_SELECTOR).length, 0);
+  });
+
+  test('user cannot trigger modal when deselecting an org with no projects selected on manage custom scopes page', async function (assert) {
+    instances.role.update({ grant_scope_ids: [instances.scopes.org.id] });
+    await visit(urls.manageScopes);
+
+    await click(`[href="${urls.manageCustomScopes}"]`);
+    await click(SCOPE_CHECKBOX_SELECTOR('org', instances.scopes.org.id));
+
+    assert.dom(REMOVE_ORG_MODAL).doesNotExist();
+
+    await click(SAVE_BTN_SELECTOR);
+
+    assert.strictEqual(currentURL(), urls.manageScopes);
+    assert.dom(BUTTON_ICON_SELECTOR).isVisible();
+
+    await click(SAVE_BTN_SELECTOR);
+
+    assert.strictEqual(currentURL(), urls.roleScopes);
+    assert.strictEqual(findAll(TABLE_ROW_SELECTOR).length, 0);
```

Comment:
I think we can do the same thing here with `assert.dom(TABLE_ROW_SELECTOR).exist({ count: 0 })`. I know we have used it in past work, however I can see that it isn't as clear. It makes me wonder if `doesNotExist()` would also work and be more clear.
```suggestion
    assert.dom(TABLE_ROW_SELECTOR).exist({ count: 0 });
```